Xamarin.Forms.Platform.Android/CollectionView/ObservableItemsSource.cs

```diff
@@ -1,23 +1,24 @@
 using System;
-using System.Collections;
+using System.Collections.Generic;
 using System.Collections.Specialized;
+using System.Linq;
```

Comment:
We try to avoid Linq.

Reply:
Updated. Now, if `IList` is used, exactly the same thing we were doing is done. In case of using `IEnumerable`, for example, to obtain the Count we iterate the items (there is no other option) but we avoid using Linq.